### PR TITLE
MF-635: Biometrics and Vitals widgets toggle buttons

### PR DESCRIPTION
**Pull request by denniskigen**
## Requirements

- [x] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [x] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Summary

This PR modifies the table/chart toggles on the Biometrics and Vitals widgets to match the provided [design](https://zpl.io/VDNgm9g).

## Screenshots

Before:

Screenshot 2021-09-13 at 16 03 21

After:

Screenshot 2021-09-13 at 15 48 45

### CODE REVIEW

**Comment by denniskigen** [comment]
Just noticed this work overlaps in part with https://github.com/openmrs/openmrs-esm-patient-chart/pull/339. I would argue, nonetheless, that this covers more of the gamut of work involved in this feature.

File: packages/esm-patient-biometrics-app/src/biometrics/biometrics-overview.scss [hunk 1]
```diff
@@ -16,6 +16,20 @@
...
+.biometricsHeaderActionItems {
+  align-items: center;
+  display: flex;
+  flex: 1 1 0%;
+  justify-content: end;
+}
+
+.divider {
+  width: 1px;
+  height: 16px;
```
Comment (FlorianRappl) [inline packages/esm-patient-biometrics-app/src/biometrics/biometrics-overview.scss:28]: Should this be 1rem?

File: packages/esm-patient-biometrics-app/src/biometrics/biometrics-overview.scss [hunk 2]
```diff
@@ -29,10 +43,57 @@
   margin: 0 $spacing-03;
 }
 
+.toggle {
+  padding: 8px;
```
Comment (FlorianRappl) [inline packages/esm-patient-biometrics-app/src/biometrics/biometrics-overview.scss:47]: Should this be 0.5rem?

File: packages/esm-patient-biometrics-app/src/biometrics/biometrics-overview.scss [hunk 3]
```diff
@@ -29,10 +43,57 @@
...
+    border-color: $interactive-01;
+  }
+
+  &:global(.bx--tooltip__trigger:active:not([disabled])) {
+    border-color: $interactive-01;
+  }
+
+  &:global(.bx--btn--tertiary:active) {
+    border-color: $interactive-01;
+    background-color: #edf5ff;
```
Comment (FlorianRappl) [inline packages/esm-patient-biometrics-app/src/biometrics/biometrics-overview.scss:61]: Are these colors defined in our variables?

File: packages/esm-patient-biometrics-app/src/biometrics/biometrics-overview.scss (hunk 1 above)
Comment (denniskigen) [reply packages/esm-patient-biometrics-app/src/biometrics/biometrics-overview.scss:28]: Yes!

File: packages/esm-patient-biometrics-app/src/biometrics/biometrics-overview.scss (hunk 2 above)
Comment (denniskigen) [reply packages/esm-patient-biometrics-app/src/biometrics/biometrics-overview.scss:47]: Correct

File: packages/esm-patient-biometrics-app/src/biometrics/biometrics-overview.scss (hunk 3 above)
Comment (denniskigen) [reply packages/esm-patient-biometrics-app/src/biometrics/biometrics-overview.scss:61]: Not this shade of blue. I raised this issue with Ciaran earlier and we've planned to work on standardising the colour palette so we have consistent naming and a comprehensive colour library in our style guide. There's not much I can do until that effort is complete.

**Review by FlorianRappl — APPROVED** [review]

**Pull request merged**